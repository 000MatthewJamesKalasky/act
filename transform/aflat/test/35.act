
defproc logic (bool in; bool a[2])
{
  spec {
    hazard(*)
  }
}

logic l;
