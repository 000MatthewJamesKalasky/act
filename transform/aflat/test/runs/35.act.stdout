hazard("l.a[0]","l.a[1]")
hazard("l.in")
